tests/resources/functional/snowflake/test_nullsafe_eq/test_nullsafe_eq_1.sql

```diff
@@ -0,0 +1,6 @@
+
+-- snowflake sql:
+SELECT A.COL1, B.COL2 FROM TABL1 A JOIN TABL2 B ON A.COL1 <=> B.COL1;
```

Comment:
Make a sperate PR for this since this is transpiler changes

Reply:
Done. #280